enclave-runtime/src/rpc/worker_api_direct.rs

```diff
@@ -113,8 +114,9 @@ where
 	// state_getMetadata
 	let state_get_metadata_name: &str = "state_getMetadata";
 	io.add_sync_method(state_get_metadata_name, |_: Params| {
-		let parsed = "world";
-		Ok(Value::String(format!("hello, {}", parsed)))
+		let metadata = Runtime::metadata();
+		let json_value = RpcReturnValue::new(metadata.encode(), false, DirectRequestStatus::Ok);
```

Comment:
Doesn't `Runtime::metadata` already return an encoded blob? Why do encode twice?

I know you seem to have tested it in the CLI. I am just confused, why this is needed.

Reply:
You're right, I will change it